config/routes.rb

```diff
@@ -55,6 +55,7 @@
 
     get '/users/me', to: 'users#show'
     get '/users/me/edit', to: 'users#edit'
+    get '/users/sign_out', to: 'devise/sessions#destroy'
```

Comment:
Is adding this route related to the issue with the gate keeper, or should it be put into another PR?